src/main/java/Bobble.java

```diff
@@ -0,0 +1,104 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Bobble {
+    public static void main(String[] args) {
+        start();
+        Scanner input = new Scanner(System.in);
+        String userInput = input.nextLine();
+        Task[] tasklist = new Task[100];
+        int taskCount = 0;
+        String command;
+
+        while (!userInput.equals("bye")) {
+            String[] UserInputs = splitUserInput(userInput);
+            command = UserInputs[0];
+            switch (command) {
```

Comment:
Using a switch here makes it really clear to read your code. Well done.

Reply:
I agree, I like how you used a `switch` to make reading cases easier!